jerry-main/main-utils.c

```diff
@@ -44,6 +44,129 @@ main_register_global_function (const char *name_p, /**< name of the function */
   jerry_release_value (result_val);
 } /* main_register_global_function */
 
+#if defined (JERRY_TEST262) && (JERRY_TEST262 == 1)
```

Comment:
What about doing this without a define? This is a simple test program, I don't see any issue to always have this comannd line feature.

Reply:
This question still stands.